Main.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <script src="https://kit.fontawesome.com/15d46053c4.js" crossorigin="anonymous"></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <title>Auto-Conference</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
- [optional] when switching from the mobile menu to the desktop version, the close icon should also disappear
- [optional] kindly align your logo with the main header and headline text.
![Screenshot (1223)_LI](https://github.com/Godedict/Auto-Conference/assets/65298048/ed59694e-276f-4006-92c0-c581e5e52f0d)

Reply:
I viewed this error on several screens and it appears  to work fine